package/contents/ui/main.qml

```diff
@@ -316,39 +348,38 @@ Item {
 				// See: http://qt-project.org/doc/qt-5/qml-qtquick-wheelevent.html#angleDelta-prop
 				while (wheelDelta >= 120) {
 					wheelDelta -= 120
-					widget.performMouseWheelUp()
+					root.performMouseWheelUp()
 				}
 				while (wheelDelta <= -120) {
 					wheelDelta += 120
-					widget.performMouseWheelDown()
+					root.performMouseWheelDown()
 				}
 				wheel.accepted = true
 			}
 		}
 
-		PlasmaCore.ToolTipArea {
-			anchors.fill: parent
-			subText: output.text
-			enabled: output.truncated
-		}
-
 		Text {
 			id: output
 			width: parent.width
 			height: parent.height
 
-			text: widget.outputText
+			PlasmaCore.ToolTipArea {
+				anchors.fill: parent
+				mainText: root.tooltipText
+				enabled: root.tooltipText
+			}
+
+			text: root.outputText
 
 			color: config.textColor
 			style: config.showOutline ? Text.Outline : Text.Normal
 			styleColor: config.outlineColor
 
-			linkColor: theme.linkColor
+			linkColor: Kirigami.Theme.linkColor
 			onLinkActivated: Qt.openUrlExternally(link)
 
-			font.pointSize: -1
-			font.pixelSize: plasmoid.configuration.fontSize * units.devicePixelRatio
-			font.family: plasmoid.configuration.fontFamily || theme.defaultFont.family
+			font.pointSize: plasmoid.configuration.fontSize
```

Comment:
`Screen.devicePixelRatio`

Reply:
> https://github.com/aricaldeira/plasma-applet-commandoutput/commit/9ea7d690df440e7c6ec53f01f9ed33b92246b8e0 @guss77: Yea, I figured later that the problem was that units no longer exists. But isn't using points for fonts better anyway? All the other font configurations in Plasma do points.

https://doc.qt.io/qt-6/qfont.html#pixelSize

`font.pointSize` is a real, so the config would need to support decimal points for the size.

Also the two sizes are very different which would create issues with upgrades:  
https://stackoverflow.com/questions/65105345/need-help-on-font-pointsize-and-font-pixelsize

Also if the scaling of the text and the GUI elements is different, then you might run into text wrapping when it's scaled.

![2024-03-08___14-55-13](https://github.com/Zren/plasma-applet-commandoutput/assets/416367/b1456b56-71fe-438e-9b64-4ac693394dee)